resources/public/api/conf/2.0/application.raml

```diff
@@ -0,0 +1,70 @@
+#%RAML 2.0
```

Comment:
Change to RAML 1.0 - this dictates the version of RAML to use, not the package it's in (ie like Python 2.7 vs 3.6 etc)